tracer/test/Datadog.Trace.Tests/Propagators/DistributedPropagatorTests.cs

```diff
@@ -18,9 +19,10 @@ public class DistributedPropagatorTests
     private const ulong SpanId = 2;
     private const int SamplingPriority = SamplingPriorityValues.UserReject;
     private const string Origin = "origin";
-    private const string RawTraceId = "1";
-    private const string RawSpanId = "2";
-    private const string PropagatedTags = "key1=value1;key2=value2";
+    private const string RawTraceId = "1a";
```

Comment:
Note to reviewers: changed these values to ensure we're not just reading `TraceId` and `SpanId`